_i18n/es/downloads.md

```diff
@@ -3,8 +3,7 @@
 
 <div class="container description" markdown="1">
 
-If you need help choosing the correct application, please click [here](https://www.reddit.com/r/Monero/comments/64b5lf/what_is_the_best_monero_wallet/) for a quick answer, then select the appropriate release for your operating system below.
-Note: the SHA256 hashes are listed by the downloads for convenience, but a GPG-signed list of the hashes is at [getmonero.org/downloads/hashes.txt](https://getmonero.org/downloads/hashes.txt) and should be treated as canonical, with the signature checked against the appropriate GPG key in the source code (in /utils/gpg_keys).
+Si necesita ayuda para elegir la aplicación correcta, haga clic aquí para obtener una respuesta rápida, luego seleccione la versión apropiada para su sistema operativo. Nota: para mayor comodidad los hashes SHA256 se enumeran por las descargas, pero una lista GPG firmada de los hash está en getmonero.org/downloads/hashes.txt y debe tratarse como canónico, con la firma marcada con la clave GPG apropiada en el código fuente (en /utils/gpg_keys).
```

Comment:
There's been some discussion going on regarding the formal vs informal Spanish, and I think we've agreed to go with the informal. Also, I think this formatting is a little off compared to the English one, if you haven't looked at how to do Markdown, you can [learn here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).